release notes/v0.25.0.md

```diff
@@ -0,0 +1,118 @@
+k6 v0.25.0 is here! :tada:
+
+This release contains mostly bug fixes, though it also has a few new features, enhancements, and performance improvements. These include HTTP request compression, `brotli` and `zstd` support, massive VU RAM usage and initialization time decreases, support for importing files via `https` and `file` URLs, and opt-in TLS 1.3 support.
+
+Thanks to @THoelzel, @matlockx, @bookmoons, @cuonglm, and @imiric for contributing to this release!
+
+## New features and enhancements!
+
+### HTTP: request body compression + brotli and zstd decompression (#989, #1082)
+
+Now k6 can compress the body of any HTTP request before sending it (#989). That can be enabled by setting the new `compression` option in the [`http.Params` object](https://docs.k6.io/docs/params-k6http). Doing so will cause k6 to transparently compress the supplied request body and correctly set both `Content-Encoding` and `Content-Length`, unless they were manually set in the request `headers` by the user. The currently supported algorithms are `deflate`, `gzip`, `brotli` and `zstd`, as well as any combination of them separated by commas (`,`).
+
+k6 now also transparently decompresses `brotli` and `zstd` HTTP responses for remote servers - previously only `deflate` and `gzip` were supported. Thanks, @imiric! (#1082)
```

Comment:
"*from* remote servers"? although I think you can drop it as well ... There are no other responses but from remote servers :)

Reply:
yeah, it was supposed to be `from,` but I'll drop the part